### DISCUSSION TITLE
Segment.tags not found data from es

### DISCUSSION THREAD

**Discussion by QQQJoker** [apache/skywalking#11260] [Q&A]
when I debug oap-server,  run to   SegmentDispatcher --> RecordStreamProcessor.getInstance().in(segment); segment has tags data,but i can't found tags data by es client . anyone konws how does it work ?

**Comment by wu-sheng** [comment]
Only pre-defined searchable tags would exist in the final storage.

**Comment by zhangzx1996** [comment]
Because the default es index mapping set exclude the field tag, you should change the mapping define
